internal/pkg/initialize/workload.go

```diff
@@ -215,7 +213,7 @@ func (w *WorkloadInitializer) initService(props *ServiceProps) (string, error) {
 		manifestExists = true
 		manifestPath = e.FileName
 	}
-	manifestPath, err = relPath(manifestPath)
+	manifestPath, err = w.Ws.Rel(manifestPath)
```

Comment:
I think so - the path seems to escape out of the function instead of just being displayed! Looking at it again, though, the path is also displayed to the user so I'll clean that up too so that the user gets the right current directory-relative path and the caller gets the correct workspace-relative path.